src/braket/circuits/noises.py

```diff
@@ -79,9 +80,15 @@ def __init__(self, probability: float):
             ascii_symbols=["BF({:.2g})".format(probability)],
         )
 
-    def to_ir(self, target: QubitSet):
+    def _to_jaqcd(self, target: QubitSet):
         return ir.BitFlip.construct(target=target[0], probability=self.probability)
 
+    def _to_openqasm(
+        self, target: QubitSet, serialization_properties: OpenQASMSerializationProperties
+    ):
+        target_qubit = serialization_properties.qubit_reference_format.format(int(target[0]))
```

Comment:
I think I would just have a method in the properties called "format_target(target:Qubit)" or  "format_target(target:QubitSetInput)"  that does the conversion for you.

Reply:
ok, added.